charts/rancher-csp-adapter/0.1.0+up0.1.0-rc2/values.yaml

```diff
@@ -0,0 +1,19 @@
+debug: true
+
+image:
+  repository: rancher/rancher-csp-adapter
+  tag: v0.1.0-rc2
+  imagePullPolicy: IfNotPresent
+
+global:
+  cattle:
+    systemDefaultRegistry: ""
+  hostNetwork: false
+
+tolerations: []
+

```

Comment:
needs the namespace default

Reply:
I can set this in the helpers, but realistically speaking I don't want to expose this option to the user. Rancher needs this in the right namespace for supportconfig functionality, so I don't think we should make this customizable.